lib/ae_page_objects/elements/collection.rb

```diff
@@ -41,7 +41,7 @@ def each
     end
 
     def size
-      node.all(:xpath, item_xpath, options).size
+      node.all(:xpath, item_xpath, options.merge(wait: false)).size
```

Comment:
I imagine this may cause some failures that will need to be updated to use the `assert_eventually` assertions.

Reply:
In Capybara 3.0 `Element#all` was changed to wait by default for at least one element.

The line here just maintains existing behavior by making it not wait.  The `Collection#size` method needs to return the current size, whether it is 0 or above.